codecarbon/emissions_tracker.py

```diff
@@ -75,6 +75,9 @@ def __init__(
         elif cpu.is_rapl_available():
             logger.info("CODECARBON : Tracking Intel CPU via RAPL interface")
             self._hardware.append(CPU.from_utils(self._output_dir, "intel_rapl"))
+        
+        if not self._hardware:
+            raise ValueError('Hardware list empty - no CPU/GPU found.')
```

Comment:
@acatovic any error / log command messages should be written like `"CODECARBON : Hardware list empty - no CPU/GPU  tracking mode found."`

Reply:
Understood, thank @kngoyal